src/embedding.jl

```diff
@@ -0,0 +1,172 @@
+"""
+    embedGigaSOM(som::Som, data::DataFrame, k, adjust, smooth)
+
+Return a data frame with X,Y coordinates of EmbedSOM projection of the data.
+
+# Arguments
+- `som`: a trained SOM
+- `data`: Array or DataFrame with the data.
+- `k`: number of nearest neighbors to consider (high values get quadratically slower)
+- `adjust`: position adjustment parameter (higher values avoid non-local approximations)
+- `smooth`: approximation smoothness (the higher the value, the larger the neighborhood of approximate local linearity of the projection)
+
+Example:
+
+Produce a 2-column matrix with 2D cell coordinates:
+
+```
+e = embedGigaSOM(som, data)
+```
+
+Plotting is best done using histograms; e.g. with Gadfly:
```

Comment:
Is this description of plotting related to `embedGigaSOM`? Maybe this would fit better in a tutorials and or the visualization folder